tests/testutils/utils.go

```diff
@@ -53,16 +54,34 @@ current-context: test-context`
 
 func RunK8sApiserver(etcdIp string) *containers.Container {
 	return containers.Run("st-apiserver",
+		"-v", os.Getenv("PRIVATE_KEY")+":/private.key",
 		fmt.Sprintf("%s", os.Getenv("HYPERKUBE_IMAGE")),
 		"/hyperkube", "apiserver",
 		"--service-cluster-ip-range=10.101.0.0/16",
 		"--authorization-mode=AlwaysAllow",
 		"--insecure-port=8080",
 		"--insecure-bind-address=0.0.0.0",
 		fmt.Sprintf("--etcd-servers=http://%s:2379", etcdIp),
+		"--service-account-key-file=/private.key",
 	)
 }
 
+func RunK8sControllerManager(apiserverIp string) *containers.Container {
+	c := containers.Run("st-controller-manager",
+		"-v", os.Getenv("PRIVATE_KEY")+":/private.key",
+		fmt.Sprintf("%s", os.Getenv("HYPERKUBE_IMAGE")),
+		"/hyperkube", "controller-manager",
+		fmt.Sprintf("--master=%v:8080", apiserverIp),
+		"--min-resync-period=3m",
+		"--allocate-node-cidrs=true",
+		"--cluster-cidr=192.168.0.0/16",
+		"--v=5",
+		"--service-account-private-key-file=/private.key",
+	)
+	time.Sleep(10 * time.Second)
```

Comment:
Can we avoid a hard coded 10s sleep here?

Reply:
Yes, of course.  This was - obviously, I thought - just a stopgap while I was asking for more high level thoughts about this change.